docs/release-notes/index.md

```diff
@@ -6,10 +6,45 @@ description: Here you'll find a chronological list of the newest Sumo Logic feat
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-| | |
-|:---|:---|
-| [**Service Release Notes**](/release-notes-service)<br/><a href="https://help.sumologic.com/release-notes-service/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a> | Latest features and bug fixes for our apps, alerts, security, search, observability, data collectors, and more. |
-| [**Cloud SIEM Release Notes**](/release-notes-cse)<br/><a href="https://help.sumologic.com/release-notes-cse/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a> | Information about new and enhanced features, updated content (like rules, log mappers and parsers), bug fixes, and other important announcements for Cloud SIEM. |
-| [**Cloud SOAR Release Notes**](/release-notes-csoar)<br/><a href="https://help.sumologic.com/release-notes-csoar/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a> | The latest news about CSOAR, like new features, bug fixes, changes to the application, and other important announcements for Cloud SOAR. |
-| [**Collector Release Notes**](/release-notes-collector)<br/><a href="https://help.sumologic.com/release-notes-collector/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a> | New Collector features and relevant bug fixes for each release. To use new Collector features mentioned here, you'll need to upgrade your Installed Collector software in one of the following ways:<ul><li> [Using Static URLs](/docs/send-data/installed-collectors/collector-installation-reference/download-collector-from-static-url)</li><li> [Using Sumo Logic](/docs/send-data/collection/upgrade-collectors)</li><li> [Using the Command Line](/docs/send-data/collection/upgrade-collectors)</li><li> [Using Collector Management API](/docs/api/collector-management/upgrade-downgrade-collectors)</li></ul> |
-| [**Developer Release Notes**](/release-notes-developer)<br/><a href="https://help.sumologic.com/release-notes-developer/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a> | New features and changes to our APIs, Collector management, and Live Tail CLI. |
+<a href="https://help.sumologic.com/release-notes-service/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a>
```

Comment:
Can you move this linked RSS icon to the Service Release Notes card? And do the same with the other categories.